t/compile.t

```diff
@@ -18,11 +18,8 @@ map {
     }, catfile($thisdir, $_));
 } qw(../bin ../lib);
 
-for (@tests) {
-    my $output = `perl -I "$libpath" -c $_ 2>&1`;
-    my $ok = ($? >> 8 == 0);
-    print $output unless $ok;
-    ok($ok, "$_ syntax check");
+foreach my $file (@tests) {
+    require_ok $file;
```

Comment:
I tried this locally, and had a few issues with the suggested code.

First, it doesn't work for me because the library path is not initialized properly. Note that previous code explicitly used `$libpath` for this purpose.

Second, `require_ok` is good for modules, but won't work for scripts (files with .pl and .t extension). `t/compile.t` is supposed to do a syntax check on all Perl files. While testing this, I realized the old code didn't pick up .pl and .t files properly. Now it does — see https://github.com/evernote/serge/commit/b4ba4a64b90e92b333eabacb48855924e18dedce. Try rebasing your PR against master and comparing the old and new code.

It would probably make sense to run `require_ok` against files with .pm extension, provided it would work correctly under Windows.